deploy/crs/v2/snippets/auth_ldap_without_properties_file.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: rhpam-ldap
+  annotations:
+    consoleName: snippet-rhpam-ldap-without-properties-file
+    consoleTitle: Configure LDAP
+    consoleDesc: Use this snippet to configure ldap opts
```

Comment:
Please make this description more specific so it's clear what are the differences between auth_ldap snippet and aith_ldap_without_properties_file snippet